api/src/handlers/configurations.js

```diff
@@ -1,11 +1,13 @@
 import λ from './utils/λ';
+import checkEventPermission from './utils/checkEventPermission';
 
 import getConfiguration from '../domain/configurations/get';
 import getHistory from '../domain/configurations/history';
 import addConfiguration from '../domain/configurations/add';
 
 const create = λ(async (event) => {
     const { id: projectId, environmentName, configName, tagName } = event.pathParameters;
+    await checkEventPermission(event, projectId, 'write');
```

Comment:
I would find it cleaner to have an `extractToken` then a `checkPermission` with the right token.
Handlers are here to do all the linking between components and it's their and only their role to extract the http request to a domain-understandable set of parameters. Also, the extraction/transformation part should IMO be domain unaware.